docs/_data/swagger.yml

```diff
@@ -209,6 +255,24 @@ paths:
                 type: string
                 x-go-name: UserID
             x-go-package: github.com/layer5io/meshery/models
+          examples:
```

Comment:
Cool. These are looking good. Do you suppose that we could have an example of a pattern that was uploaded to Meshery from GitHub? // @Chadha93

Reply:
name: IstioSM
version: 1.0.1
services:
  istio:
    type: IstioMesh
    namespace: istio-system
    settings:
      version: 1.9.1
    traits:
      mTLS:
        policy: mutual
        namespaces:
          - istio-test
      automaticSidecarInjection:
        namespaces:
          - default
          - istio-test
​
  grafana:
    type: GrafanaIstioAddon
    namespace: istio-system
    dependsOn:
      - istio
      - prometheus
​
  prometheus:
    type: PrometheusIstioAddon
    namespace: istio-system
    dependsOn:
      - istio
      
  prometheus2:
    type: PrometheusIstioAddon
    namespace: istio-system
    dependsOn:
      - istio